script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -513,57 +605,69 @@ She really is surprised. Is it because in the end, her prey displayed some human
 [sha:b8d2ef6d95728ca1f1d76731658f81640f12a2d5]{
 -- Page 461, Offset 19420.
 -- そんな、人間であれば誰もが知っている常識を、“何故”と言われた事に驚いたのだ。
-She's surprised because she was asked something any human should know.
+She's surprised because he asked her something any human would know as a matter of course.
```

Comment:
"as a matter of course" at the end here is kinda clunky.

> something that would be common sense to any human
something that any human would know.
something that would be obvious to [any|a] human